Frameworks/AddressBook/ABContactInternal.mm

```diff
@@ -28,4 +28,18 @@ - (id)initWithContact:(WACContact*)contact {
     return self;
 }
 
++ (NSDate*)convertDate:(WACContactDate*)date {
```

Comment:
I think it would make more sense to be a helper function unattached to a class or a category method for NSDate

---
In reply to: [75164209](https://github.com/Microsoft/WinObjC/pull/884#discussion_r75164209) [](ancestors = 75164209,75146840)